ftw/sliderblock/browser/resources/sliderblock.js

```diff
@@ -1,9 +1,13 @@
-(function(global, $) {
-
-  "use strict";
-
+(function (root, factory) {
+    if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define(['jquery', 'slider'], factory);
+    } else {
+        // Browser globals
+        factory(root.jQuery, root.Slider);
+    }
+}(typeof self !== 'undefined' ? self : this, function ($, Slider) {
```

Comment:
Is `typeof self !== 'undefined' ? self : this` something that we should do be default and that should be updated in the documentation at https://devdocs.4teamwork.ch/plone/javascript/ftw-resource-bundling-plone5/#transform-our-javascript-in-the-product ?